CmdStarter.Tests.Commands/Erroneous/AutoComplete/GenericWrongFactory.cs

```diff
@@ -0,0 +1,16 @@
+﻿using com.cyberinternauts.csharp.CmdStarter.Tests.Commands.Interfaces;
+using NUnit.Framework;
+
+namespace com.cyberinternauts.csharp.CmdStarter.Tests.Commands.Erroneous.AutoComplete
+{
+    public class GenericWrongFactory : IHasException, IErrorRunner, IGetDefault<GenericWrongFactory>
```

Comment:
Isn't a bit weird that the classes in `Erroneous.AutoComplete` don't inherit from `StarterCommand` like all the others? (I know, it is not mandatory... but just a bit weird)

Reply:
I don't think it's a huge deal as this still could be considered controlled errors.

Although these don't need to be separated from the rest as it can only be invoked manually.

What do you think? Shall Errorous only keep classes that can throw exceptions without us manually calling it or anything that has controlled errors?

Reply:
Not sure to understand the difference between "controlled errors" and "throw exceptions without us manually calling it". Isn't all the same? An exception is thrown in both cases?

Anyhow, the classes are in the proper namespace.